lib/ui-common/src/ui.test.ts

```diff
@@ -11,7 +11,7 @@ describe('prepareAnnotationsForPresentation', () => {
                         start: { line: 3, character: 4 },
                         end: { line: 5, character: 6 },
                     },
-                    item: { title: '1' },
+                    item: { title: '1', kind: 'file' },
```

Comment:
kind is optional, did this need updating? I don't think kind plays a part in annotations right?

Reply:
Yeah, I earlier kept it required. Removed the changes.